### ISSUE TITLE
Broken URL notifications not working on Craft 4

### ISSUE THREAD

**Issue by steverowling** [wrav/oembed#125]
If I enable notifications in the plugin on Craft 4.2.4, I get the following error in `phperrors.log`

```
Declaration of wrav\oembed\jobs\BrokenUrlNotify::execute($queue) must be compatible with craft\queue\JobInterface::execute($queue): void in /var/www/html/vendor/wrav/oembed/src/jobs/BrokenUrlNotify.php on line 22
```

It looks like you need to add a return type of `: void` to the `execute()` function on line 22.

**Comment by thijskaspers** [comment]
I'm getting this error again with Craft 4.3.3 and oEmbed 2.2.2, so doesn't seem to be fixed?

**Comment by tomfischerNL** [comment]
Hi @steverowling, I'm having the same issue when leaving a field empty.

(Craft version 4.3.6.1)

**Comment by reganlawton** [comment]
I believe this was resolved in v2.3.1
